src/app/components/Recommendations/RecommendationsPromoImage/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useContext } from 'react';
+import { createSrcsets } from '#lib/utilities/srcSet';
+import buildIChefURL from '#lib/utilities/ichefURL';
+import Image from '#app/components/Image';
+import { Recommendation } from '#models/types/onwardJourney';
+import { RequestContext } from '#app/contexts/RequestContext';
+
+const RecommendationsImage = ({
+  image,
+  lazyLoad = false,
+}: {
+  image: Recommendation['image'];
+  lazyLoad?: boolean;
+}) => {
+  const { isAmp } = useContext(RequestContext);
+  const { height, width, altText, copyrightHolder, originCode, locator } =
+    image;
+
+  const imageResolutions = [70, 95, 144, 183, 240, 320, 660];
+  const { primarySrcset, fallbackSrcset } = createSrcsets({
+    originCode,
+    locator,
+    originalImageWidth: width,
+    imageResolutions,
+  });
+  const DEFAULT_IMAGE_RES = 660;
+  const src = buildIChefURL({
+    originCode,
+    locator,
+    resolution: DEFAULT_IMAGE_RES,
+  });
```

Comment:
Apologies for not spotting this in the BFF PR, but can we have the BFF returning an ichef URL, so that we don't need to worry about building the URL in Simorgh?

Reply:
Feel like this came up before in a different context and we didn't come to a conclusion on this type of logic.

My opinion is that display logic like this should live closer to the client, more so because it _feels_ like it should. but equally it would work fine coming from the server/API.

I guess it could be part of a wider refactor piece, as we'd have a bit of a mixed economy if we moved all that `createSrcSet` and `buildIChefUrl` logic to the BFF just for this.

Reply:
We have a couple of tech ticket for this: https://jira.dev.bbc.co.uk/browse/WORLDSERVICE-268 & https://jira.dev.bbc.co.uk/browse/WORLDSERVICE-262 but think these might need a wider discussion about this. We already have a confusing & mixed economy for this, so it's no worse than it already is. It would be nice to consolidate it eventually, but perhaps not as part of this work?

Reply:
Ah yes, think those were the tickets we talked about at one stage. I would like a wider convo on it to see where everyone thinks the logic should sit.

Reply:
Defintely. One for a future tech session - perhaps when we're auditing these particular tickets?